apps/unified-storybook/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": "tsconfig/base.json",
+    "compilerOptions": {
+        "types": ["vitest/globals"],
+        "baseUrl": ".",
+        "paths": {
+            "@/*": ["../../packages/mirinae/src/*"]
+        },
+        "allowImportingTsExtensions": true,
+        "noEmit": true
+    },
+    "include": [
+        "**/*.ts",
+        "**/*.js",
+        "../../packages/mirinae/**/*.@(ts|js|cjs|vue|mdx)"
```

Comment:
question: Would this section need some adjustments? I'd love to hear your thoughts on potential improvements here.